sympy/ntheory/generate.py

```diff
@@ -1005,3 +1005,34 @@ def compositepi(n):
     if n < 4:
         return 0
     return n - primepi(n) - 1
+
+def nprime(nth, kth=1):
+    """ Returns kth to nth prime numbers(both inclusive).
```

Comment:
Follow the docstring style guide. https://docs.sympy.org/latest/documentation-style-guide.html